### DISCUSSION TITLE
Deploy on save exclusions

### DISCUSSION THREAD

**Discussion by robert-strunk** [forcedotcom/salesforcedx-vscode#5792] [Ideas]
**Is your feature request related to a problem? Please describe.**
I have  [Deploy on Save](https://developer.salesforce.com/tools/vscode/en/user-guide/deploy-on-save/) enabled. When I am working on LWC unit tests, each save triggers a failing push/deploy. Its a bit distracting. 

**Describe the solution you'd like**
There may already be some mechanism to prevent that from happening but I wasn't able to find anything. I think a solid solution would be to create a new setting that allows you to specify directories that should not be included in the deploy on save functionality (similar to how a .forceignore ignores directories/files on push/pull/status)

**Describe alternatives you've considered**
Disabling deploy on save. I've also looked for existing settings/mechanisms that may address this issue.

**Comment by Ameensam1** [comment]
I have this problem too.

**Comment by wildcatgerry** [comment]
I thought that `.forceignore` entries would be respected by the extension too.
